OrbitGl/SamplingReportDataView.cpp

```diff
@@ -155,19 +155,20 @@ std::vector<const FunctionInfo*> SamplingReportDataView::GetFunctionsFromIndices
     }
   }
 
-  return std::vector<const FunctionInfo*>(functions_set.begin(), functions_set.end());
+  return functions_set;
 }
 
-std::vector<ModuleData*> SamplingReportDataView::GetModulesFromIndices(
+absl::flat_hash_set<ModuleData*> SamplingReportDataView::GetModulesFromIndices(
```

Comment:
Same here.

Reply:
Yes, otherwise, we would try to load multiple modules at once.

Reply:
Also it was a set for the functions internally already.

Reply:
Ah I see that the implementation of this didn't use a `set` first internally, unlike `GetFunctionsFromIndices`.